tests/CloseConflictingAppsTests.js

```diff
@@ -166,38 +166,43 @@ gpii.tests.conflictingApps.testDefs = [
                 "com.freedomscientific.jaws": [
                     {
                         "settings": {
-                            "Voice Profiles.ActiveVoiceProfileName": "GPII",
                             "Options.SayAllIndicateCaps": 1,
                             "Options.TypingEcho": 2
                         },
                         "options": {
                             "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\2019\\Settings\\enu\\DEFAULT.JCF"
                         }
-                    },
-
+                    }
+                ]
+            },
+            "gpii.settingsHandlers.JAWSSettingsHandler": {
+                "com.freedomscientific.jaws": [
                     {
                         "settings": {
-                            "ENU-Global.Rate": 400,
-                            "ENU-Global.Punctuation": 3,
-                            "ENU-Global.Pitch": 11,
-                            "ENU-Message.Rate": 400,
-                            "ENU-Message.Punctuation": 3,
-                            "ENU-Message.Pitch": 11,
-                            "ENU-Keyboard.Rate": 400,
-                            "ENU-Keyboard.Punctuation": 3,
-                            "ENU-Keyboard.Pitch": 11,
-                            "ENU-PCCursor.Rate": 400,
-                            "ENU-PCCursor.Punctuation": 3,
-                            "ENU-PCCursor.Pitch": 11,
-                            "ENU-JAWSCursor.Rate": 400,
-                            "ENU-JAWSCursor.Punctuation": 3,
-                            "ENU-JAWSCursor.Pitch": 11,
-                            "ENU-MenuAndDialog.Rate": 400,
-                            "ENU-MenuAndDialog.Punctuation": 3,
-                            "ENU-MenuAndDialog.Pitch": 11
+                            // NOTE: Commented until GPII-4336 is addressed
```

Comment:
It's not good to have these settings definitions in tests that really only use other parts of the solution definition.  I'd suggest much leaner test solutions with like one setting each.

Reply:
Well, on the bright side, this PR actually comments out most of all the settings from these tests. :)